distribution/build-distribution.ps1

```
@@ -0,0 +1,676 @@
+    [string] $InstallerScript =
+        "./Expressif.iss",
+
+    [string] $Version = $(
+        if ([string]::IsNullOrWhiteSpace($env:GitVersion_SemVer)) {
+            "0.0.0-local"
+        } else {
+            $env:GitVersion_SemVer
+        }
+    ),
+
+    [string[]] $Frameworks = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Frameworks' has been declared but not used.

(PSReviewUnusedParameter)